packages/generator/__tests__/assets/templates/pet.json

```diff
@@ -404,7 +404,16 @@
+  "basePdf": {
```

Comment:
The main part of what I did in this PR is that I made it so that we can set it with a new type called [BlankPdf](https://github.com/pdfme/pdfme/pull/394/files#diff-137a58655e18fd31de35b0116e6e0e215889557a42c396f63f0479c20f3bba9fR87) instead of PDF data in basePdf.